android/app/src/main/java/com/dismoi/scout/accessibility/BackgroundService.kt

```diff
@@ -115,4 +138,50 @@ class Activity : AccessibilityService() {
     browsers.add(SupportedBrowserConfig("com.android.chrome", "com.android.chrome:id/url_bar"))
     return browsers
   }
+
+  private fun createNotificationChannel() {
+    // Create the NotificationChannel, but only on API 26+ because
+    // the NotificationChannel class is new and not in the support library
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+      val importance = NotificationManager.IMPORTANCE_DEFAULT
+      val channel = NotificationChannel(CHANNEL_ID, "BACKGROUND", importance)
+      channel.description = "CHANEL DESCRIPTION"
+      val notificationManager = getSystemService(NotificationManager::class.java)
+      notificationManager.createNotificationChannel(channel)
+    }
+  }
+
+  override fun onCreate() {
+    super.onCreate()
+  }
+
+  override fun onDestroy() {
+    super.onDestroy()
+    handler.removeCallbacks(runnableCode)
+  }
+
+  override fun onStartCommand(intent: Intent, flags: Int, startId: Int): Int {
+    createNotificationChannel()
+    val notificationIntent = Intent(this, MainActivity::class.java)
+    val contentIntent = PendingIntent.getActivity(
+      this,
+      0,
+      notificationIntent,
+      PendingIntent.FLAG_CANCEL_CURRENT
+    )
+    val notification = NotificationCompat.Builder(this, CHANNEL_ID)
+      .setContentTitle("DisMoi")
+      .setContentText("DisMoi is running in background...")
+      .setSmallIcon(R.mipmap.ic_launcher)
+      .setContentIntent(contentIntent)
+      .setOngoing(true)
+      .build()
+    startForeground(SERVICE_NOTIFICATION_ID, notification)
+    return START_STICKY
+  }
+
+  companion object {
+    private const val SERVICE_NOTIFICATION_ID = 12345
```

Comment:
Should it be in *setting* or a *env* thing instead?

Reply:
What do you mean?